hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockInputStream.java

```diff
@@ -163,9 +163,13 @@ public synchronized void initialize() throws IOException {
       try {
         blockData = getBlockData();
         chunks = blockData.getChunksList();
+        LOG.debug("Block {} has block data {}", blockID, blockData);
```

Comment:
Could you add the if(LOG.isDebugEnabled) check for these newly added logs?

Reply:
the parameterized logging with a simple object reference has negligible overhead. But let's remove it. It's redundant.